pyproject.toml

```diff
@@ -26,17 +26,18 @@ authors = [
     {name="Thomas Manke"},
     {email="bioinfo-core@ie-freiburg.mpg.de"}
 ]
-requires-python = "> 3.8"
+requires-python = "> 3.9"
 dependencies = [
-    "numpy >= 2.0.0",
-    "scipy >= 0.17.0",
-    "matplotlib >= 3.5.0",
-    "pysam >= 0.14.0",
-    "numpydoc >= 0.5",
-    "pyBigWig >= 0.2.1",
-    "py2bit >= 0.2.0",
-    "plotly >= 4.9",
-    "deeptoolsintervals >= 0.1.8"
+    "numpy >= 2.0",
+    "scipy >= 1.13",
+    "matplotlib >= 3.9",
+    "pysam >= 0.23",
+    "numpydoc >= 1.8",
+    "pyBigWig >= 0.3",
+    "py2bit >= 0.3",
+    "pandas >= 2.2",
```

Comment:
is pandas a requirement now ? 
As for the minimum versions, are these correct ? (mainly mpl, scipy, pysam -> how come this changed ?)

Reply:
Unfortunately pandas is a dep for scikit-learn, which is added for the PCA analysis. For the others, those are the minimal version compatible with python 3.9, some don't have a real change (pysam, ...)